provisioner/env.go

```diff
@@ -58,6 +58,9 @@ const (
 	// NFSBackendPvcTimeout defines env name to store BackendPvcBoundTimeout value
 	NFSBackendPvcTimeout menv.ENVKey = "OPENEBS_IO_NFS_SERVER_BACKEND_PVC_TIMEOUT"
 
+	// NFSCleanUpStalePvcEnable defines env name to store the value of clean stale pvc enable
+	NFSCleanUpStalePvcEnable menv.ENVKey = "OPENEBS_IO_NFS_SERVER_CLEANUP_STALE_PVC_ENABLE"
```

Comment:
This flag essentially enables/disables garbage collection. Even though the garbage collection currently removes stale PVCs, maybe the flag should be instead called `OPENEBS_IO_NFS_SERVER_GARBAGE_COLLECTION_ENABLED`? Then in case the logic of what the garbage collector does is changed, this flag won't be misleading :)

Reply:
> Even though the garbage collection currently removes stale PVCs, maybe the flag should be instead called OPENEBS_IO_NFS_SERVER_GARBAGE_COLLECTION_ENABLED?

Do you mean you'd rather always enable stale cleanup rather than disable it completely?

>  Then in case the logic of what the garbage collector does is changed, this flag won't be misleading

Yes, when the logic of cleanup is optimized, we actually do not need to turn off recycling. Currently, only for data recovery scenarios, completely turning off can solve my problem.

Of course, it is also possible if we define some rules to avoid recycling stale pvc. For example, if certain labels are included, they will not be recycled.